configure.ac

```diff
@@ -258,6 +258,27 @@ if test "x$with_di_csvfile" == xyes; then
    BS_DI_OPT(csvfile-csv-file, CSVFILE_CSV_FILE, CSV file listing the MRT data to read, not-set)
 fi
 
+# RPKI configuration (ROAFetchLib)
+AC_MSG_NOTICE([])
+AC_MSG_NOTICE([---- RPKI support configuration ----])
+AC_ARG_WITH([rpki],
+        [AS_HELP_STRING([--without-rpki],
+        [do not compile with rpki support])],
+        [],
+        [with_rpki=yes])
```

Comment:
consider default no for `rpki` support. @alistairking what do you think?

Reply:
Definitely. 
I think we have two options:
 - if roafetchlib is installed, enable rpki by default
 - disable rpki unless explicitly requested

I'm ok with the first option iff the rpki code stays dormant unless explicitly requested at run time, which, i'm assuming is how things work at the moment.

Reply:
Okay. Now RPKI support is disabled unless explicitly requested with ```with-rpki```. I chose it because it's a bit more in line with the other options. I don't think the two approaches differ much in practice, because the ROAfetchlib is not included anyway.